docs/pages/deployment/configuration.md

```diff
@@ -42,6 +42,10 @@ Minimal log level that should be logged. Supported options:
 
 See `-loglevel` option in [FFmpeg documentation](https://ffmpeg.org/ffmpeg.html).
 
+### `LIVE_COMPOSITOR_TMP_PATH`
```

Comment:
TMP_PATH does not have any meaning, the option name should describe what they are doing e.g.

`DOWNLOAD_DIR`/`TMP_DOWNLOAD_DIR`, even `WORKING_DIR` (its very generic but at least describes the purpose).

Also, document what happens when multiple compositors start with the same path, or if a directory already exists. I would prefer if you created a subdirectory with random id in the name inside that user specified path.